src/identity/identityService.ts

```diff
@@ -0,0 +1,21 @@
+import {AxiosInstance} from 'axios'
+import {Identity} from './identity'
+
+export class IdentityService {
+	http: AxiosInstance
+
+	constructor(http: AxiosInstance) {
+		this.http = http
+	}
+
+	async getDetails(token: string) {
```

Comment:
any timeouts set for this requests or global timeouts?

Reply:
Timeouts set in applicationContext()